store/postgres_migration.go

```diff
@@ -157,30 +159,11 @@ func alterColumnStatement(provider config.StoreProviderType, tenant string, info
 	}
 }
 
-// alter a join constraint if it exists. This will first drop an existing constraint and then add it again
-// with the updated values. This is because there is not a simple way to alter a constraint in SQL
-func alterJoinStatement(tenant string, info tableInfo, uniqueInterface interface{}) ([]string, error) {
-	unique, ok := (uniqueInterface).(bool)
-	if !ok {
-		return nil, fmt.Errorf("uniqueInterface not assignable to bool: %s", uniqueInterface)
-	}
-	var statements = make([]string, 0, 1)
-	statements = append(statements,
-		"ALTER TABLE IF EXISTS "+psqlAbsTableName(tenant, info.TableName)+" DROP CONSTRAINT IF EXISTS "+info.TableName+"_"+info.ElementName+"_unique;",
-	)
-	if unique {
-		statements = append(statements,
-			"ALTER TABLE IF EXISTS "+psqlAbsTableName(tenant, info.TableName)+" ADD CONSTRAINT "+info.TableName+"_"+info.ElementName+"_unique UNIQUE ("+info.ElementName+");",
-		)
-	}
-	return statements, nil
-}
-
 // this will create a column in a table, and then if specified add the UNIQUE constraint
 func createFieldStatement(tenant string, info tableInfo, fieldInterface interface{}) ([]string, error) {
 	field, ok := (fieldInterface).(core.TableField)
 	if !ok {
-		return nil, fmt.Errorf("fieldInterface not assignable to core.TableField: %s", fieldInterface)
+		return nil, fmt.Errorf("cannot assign type %s to core.TableField", reflect.TypeOf(fieldInterface).String())
```

Comment:
❓ The canonical logging format would be `"cannot assign type to core.TableJoin: %s"`, no?